modules/quickstart-sap/pages/prepare-clients.adoc

```diff
@@ -28,7 +28,7 @@ For more information about registering {productname} with {scc}, see xref:instal
 
 
 
-== Create the Cluster
+== Configure the Cluster
```

Comment:
:heart:

Reply:
"Hosts" is a pretty confusing name to use in this context, because in SUMA terms they're actually clients. I've changed it to "Configure the Clients for Clustering", is that better?